locales/fr.json

```diff
@@ -160,36 +160,36 @@
   "prompt.keys_removed": "{0} clés retirés",
   "prompt.keystyle_flat": "Modèle plat",
   "prompt.keystyle_flat_example": "par exemple : { \"a.b.c\": \"...\" }",
-  "prompt.keystyle_nested": "Modèle emboîté",
-  "prompt.keystyle_nested_example": "par exemple : { \"a\": { \"b\": { \"c\": \"...\" } } }",
+  "prompt.keystyle_nested": "Modèle emboité",
+  "prompt.keystyle_nested_example": "par exmple : { \"a\": { \"b\": { \"c\": «...» } } }",
```

Comment:
`emboîté` to `emboité`: is it on purpose?

Reply:
same as previous.

Reply:
It's `emboîté` here, the new translation is incorrect